bigbluebutton/bigbluebutton
# chore: bump tldraw to alpha.23

### What does this PR do?

Pulls in the latest bbb/tldraw https://github.com/bigbluebutton/tldraw/releases/tag/v2.0.0-alpha.23

### Closes Issue(s)

Closes # none
Resolves a browser console error about locales


### Motivation



### How to test

The package `bbb-html5` should build and install fine. Whiteboard should work normally. No errors in the browser console after whiteboard loading.

### More
